cli-plugins/manager/candidate_test.go

```diff
@@ -0,0 +1,81 @@
+package manager
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	cliplugins "github.com/docker/cli/cli-plugins"
+	"github.com/spf13/cobra"
+	"gotest.tools/assert"
+)
+
+type mockCandidate struct {
```

Comment:
absolute nit: this is more a `fake` than a `mock` :upside_down_face:

Reply:
I will run `sed`...